<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnBlockMe</class>
 <widget class="QWidget" name="UnBlockMe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UnBlock Me</string>
  </property>
  <widget class="QTableWidget" name="gameBoardTableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="dfsPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>DFS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Execution time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0ms</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>291</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;--</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forwardPushButton">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>440</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>--&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadLevelPushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>355</y>
     <width>105</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>LoadLevel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="levelPathLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bfsPushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>BFS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gbfsPushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>GBFS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
